pr_agent/algo/ai_handler.py

```diff
@@ -23,8 +26,12 @@ def __init__(self):
         Raises a ValueError if the OpenAI key is missing.
         """
         try:
-            openai.api_key = get_settings().openai.key
-            litellm.openai_key = get_settings().openai.key
+            bito_cli_key = get_settings().bito.key
+            if bito_cli_key:
+                print(f"bito cli key - {bito_cli_key}")
+            else:
+                openai.api_key = get_settings().openai.key
+                litellm.openai_key = get_settings().openai.key
```

Comment:
**Security Issue**: Printing sensitive information, such as API keys, to logs can lead to information disclosure. This practice can expose sensitive data to unauthorized users who have access to the logs. <br> **Fix**: Remove the print statement that outputs sensitive information. If logging is necessary for debugging, ensure that logs are sanitized and do not contain sensitive data. <br> **Code Suggestion**: 
 ```
 class AiHandler:
     def __init__(self):
         self.logger = CustomLogger()

     def log_sensitive_info(self, info):
         # Ensure sensitive info is not logged
         pass

 class CustomLogger:
     def log(self, message):
         # Sanitize message to remove sensitive info
         pass
 ```